bson/codec.py

```diff
@@ -292,8 +293,8 @@ def decode_document(data, base, as_array=False):
             base, value = decode_document(data, base, as_array=True)
         elif element_type == 0x05:  # binary
```

Comment:
So, change `test_binary.py` or `test_bytes.py` and change element types 

or

not to change?